easybuild/easyconfigs/l/LAPACK/LAPACK-3.9.1-GCC-10.2.0.eb

```diff
@@ -9,7 +9,7 @@ description = """LAPACK is written in Fortran90 and provides routines for solvin
 toolchain = {'name': 'GCC', 'version': '10.2.0'}
 toolchainopts = {'pic': True}
 
-source_urls = ['https://github.com/Reference-LAPACK/lapack/archive/']
+source_urls = ['https://github.com/Reference-LAPACK/lapack/archive/refs/tags']
```

Comment:
This change doesn't make sense to me, the original URL is working fine?!

Reply:
It was 404 to me on two machines.

Reply:
The new URLs is used on the release page: https://github.com/Reference-LAPACK/lapack/releases
That is why I think it makes sense to update.